import{f as n,g as a,J as s}from"./common-03e46d7f.js";const t='{"title":"不是吧？2023了竟然还有人用Try-Catch来处理async...await异常","frontmatter":{"date":"2023-04-16","title":"不是吧？2023了竟然还有人用Try-Catch来处理async...await异常","tags":["javascript","work"],"describe":"async/await 是在 ES2017 中引入的，目的是为了让异步操作更加直观、方便，同时也解决了 Promise 的回调地狱问题。那么我们该什么时候去捕获 async/await 的异常呢？"},"headers":[{"level":2,"title":"写在前面","slug":"写在前面"},{"level":2,"title":"什么时候会请求异常","slug":"什么时候会请求异常"},{"level":2,"title":"什么情况下需要处理请求异常","slug":"什么情况下需要处理请求异常"},{"level":3,"title":"多个异步请求串行","slug":"多个异步请求串行"},{"level":3,"title":"处理异步请求的 loading 状态","slug":"处理异步请求的-loading-状态"},{"level":2,"title":"处理方法","slug":"处理方法"},{"level":3,"title":"使用 Promise 处理","slug":"使用-promise-处理"},{"level":3,"title":"await-to-js 处理函数","slug":"await-to-js-处理函数"},{"level":2,"title":"总结","slug":"总结"}],"relativePath":"docs/javascript/async-await.md","lastUpdated":1743653323512.1055}';var p={};const o=[s('',34)];p.render=function(s,t,p,e,c,l){return n(),a("div",null,o)};export{t as __pageData,p as default};
